Project.toml

```diff
@@ -5,28 +5,30 @@ version = "0.22.17"
 
 [deps]
 CRlibm_jll = "4e9b3aee-d8a1-5a3d-ad8b-7d824db253f0"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
```

Comment:
Do we use LinearAlgebra in the main IntervalArithmetic module?

Reply:
I added this because Aqua was throwing some comment, but indeed, it seems unnecessary. I've removed it in [3ca341d](https://github.com/JuliaIntervals/IntervalArithmetic.jl/pull/687/commits/3ca341d24ce9e2f5ac10b55996a3e88505b50b7c)